andela/Ah-backend-xmen
# #162725250 Setup and deploy base application to heroku

#### What does this PR do?
This PR is to set up and deploy base application to Heroku.

#### Description of the task to be completed.
Setup and deploy base application to Heroku so that other users can access the application.

#### how should this be manually tested?
- First things first, fetch this branch by running `git fetch origin ch-setup-app-and-deploy-to-heroku-162725250`
- Check into this branch by running ` git checkout -- ch-setup-app-and-deploy-to-heroku-162725250`
- Activate your virtual environment by running `source venv/bin/activate`
- Install dependencies by running `pip install -r requirements.txt`
-  View  live `staging app`  [here](https://ah-backend-staging.herokuapp.com/api/)

To run locally, you need to have a `heroku-cli` installed on your machine
   [Link on how to install heroku cli](https://devcenter.heroku.com/articles/heroku-cli)
    - run local Heroku  local server using `heroku local web`
   - navigate to `localhost:5000/api/` in your web browser

#### Relevant stories
[#162725250](https://www.pivotaltracker.com/story/show/162725250)
